src/cdk/overlay/overlay-ref.ts

```diff
@@ -471,7 +471,10 @@ export class OverlayRef implements PortalOutlet, OverlayReference {
 
     coerceArray(cssClasses).forEach(cssClass => {
       // We can't do a spread here, because IE doesn't support setting multiple classes.
-      isAdd ? classList.add(cssClass) : classList.remove(cssClass);
+      // Also trying to add an empty string to a DOMTokenList will throw.
+      if (cssClass !== '') {
```

Comment:
Could this just be `if (cssClass)`?

Reply:
It's super edge-case-ey, but it would technically exclude a class called `false`.

Reply:
I don't believe that's true; the string `'false'` is truthy

Reply:
I mean somebody passing in `false` as a boolean.

Reply:
I would actually expect doing nothing in that case over adding a `'false'` CSS class

Reply:
Changed.